app/extensions/brave/content/scripts/blockCanvasFingerprinting.js

```diff
@@ -7,167 +7,187 @@
  * Chameleon <https://github.com/ghostwords/chameleon>, Copyright (C) 2015 ghostwords
  * Privacy Badger Chrome <https://github.com/EFForg/privacybadger>, Copyright (C) 2015 Electronic Frontier Foundation and other contributors
  */
+Error.stackTraceLimit = Infinity // collect all frames
```

Comment:
this file was refactored so that `blockWebRTC` is a separate function. despite the large diff, the only material change is that `blockWebRTC` is called if `chrome.contentSettings.torEnabled == 'block'`

Reply:
Maybe this could be done in two separate commits to make the audit trail clearer -- one commit with no functional change to refactor, and another commit with the one-line functional change?

Reply:
not a bad idea, but it goes against our custom of "1 commit per PR" 🤷‍♀️

Reply:
So that's why there's been so much squashing around here that makes my historian skin crawl!